homeassistant/components/motionblinds_ble/entity.py

```diff
@@ -23,8 +23,18 @@ class MotionblindsBLEEntity(Entity):
     _device: MotionDevice
     config_entry: ConfigEntry
 
-    def __init__(self, device: MotionDevice, entry: ConfigEntry) -> None:
+    def __init__(
+        self,
+        device: MotionDevice,
+        entry: ConfigEntry,
+        unique_id_suffix: str | None = None,
+    ) -> None:
         """Initialize the entity."""
+        self._attr_unique_id: str = (
```

Comment:
@starkillerOG I also moved the unique_id to the base class, with the possibility of adding a suffix for the other entities. Is something like `ff-ff-ff-ff-ff-ff_battery` fine as a unique ID for the other entities?

Reply:
Yes that is good.
Although normally this is done in the key of the entity description, but this is fine.

Reply:
Awesome! Then I think the integration is ready for someone else to review it